docs/aserto-console/manage-organizations.mdx

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_label: Managing Organizations
+---
+
+# Managing Organizations
+
+:::caution note
+When first signing up to Aserto, you'll be provisioned with a personal account, which doesn't allow inviting other members. During the free Beta period, you can open a single organization and invite team members into it.
```

Comment:
you can create organizations and invite...

Reply:
Shouldn't the limit be mentioned?

Reply:
the limit is 5. I don't think we need to mention it here.